spanner/api/Spanner.Samples/AddAndDropDatabaseRoleAsync.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Copyright 2023 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START spanner_add_and_drop_database_role]
+
+using Google.Cloud.Spanner.Data;
+using System.Threading.Tasks;
+
+public class AddAndDropDatabaseRoleAsyncSample
+{
+    public async Task AddDatabaseRoleAsync(string projectId, string instanceId, string databaseId, string databaseRole)
+    {
+        string connectionString = $"Data Source=projects/{projectId}/instances/{instanceId}/databases/{databaseId}";
+        string createRoleDDlStatement = $"CREATE ROLE {databaseRole}";

```

Comment:
If they are on the same region tag, be consistent with variable naming.

Reply:
Address this please.

Reply:
Could you please elaborate this a bit more.

Reply:
One variable is named `createRoleDDlStatemet` and the other one is named `dropDatabaseRoleStatement`. The names are not equivalent and that may lead to confusion, in particular because they are on the same region tag. Better to name them:
`createRoleStatement`
`deleteRoleStatement`